scripts/after_prepare.js

```diff
@@ -102,9 +102,9 @@ function copyKey(platform, callback) {
 }
 
 function getValue(config, name) {
-    var value = config.match(new RegExp('<' + name + '>(.*?)</' + name + '>', 'i'));
-    if (value && value[1]) {
-        return value[1]
+    var value = config.match(new RegExp('<' + name + '(.*?)>(.*?)</' + name + '>', 'i'));
```

Comment:
what happens here when no short name is added?  If this regex is not matched, shouldn't we default to the previous logic?